src/Workspaces/Remote/Core/ProjectSystem/WorkspaceProjectCreationInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem
+{
+    public record WorkspaceProjectCreationInfo(string Language, string DisplayName, string? FilePath, IReadOnlyDictionary<string, string> BuildSystemProperties)
+    {
+    }
```

Comment:
```suggestion
    public record WorkspaceProjectCreationInfo(string Language, string DisplayName, string? FilePath, IReadOnlyDictionary<string, string> BuildSystemProperties)
    {
    }
```

record-struct?

Reply:
@CyrusNajmabadi Your diff there was supposed to add a 'record struct' there? Otherwise I think the edit is identical...?

Reply:
weird.  i was removing the braces :)

Reply:
Ah, you did something wrong there. :smile: